python/py-precis-i18n/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        byllyfish precis_i18n 1.0.4 v
+name                py-precis-i18n
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         PRECIS framework for Unicode
+long_description    Implementation of PRECIS framework for Unicode to be used with usernames & passwords
+
+checksums   rmd160  e3cabcc6e8fe8644cefe867ff9a871cf2c287f44 \
+            sha256  2c177768cc1d3a3ddff0b53f0143ab1e5c2fb1c89106a6d2d1b89b7c68fd9c9f \
+              size  231854
```

Comment:
please align the checksums porperly

Reply:
I’d say it is debatable, but I won’t argue over alignments. Done.

Reply:
it's not debatable - it's what the MacPorts' project ones and the alignment should conform to the modeline (what you had doesn't)